Marlin/Conditionals_LCD.h

```diff
@@ -92,7 +101,7 @@
     #endif
   #endif
 
-  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
+  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER) || ENABLED(ANET_FULL_GRAPHICS_LCD)
```

Comment:
To update this little thread, I did keep the conditionals changes, leaving in place the original pins code. I get what you're saying about the A6 being the main and non-exotic piece of hardware. Maybe for the sake of A8 pinouts we should leave a comment about the situation and tell them what to change — or point them to the website, if it's o'erlong.